src/components/color-picker/color-picker.css

```
@@ -1,25 +1,30 @@
+    color-scheme: light;
 }
 
 :global(.Popover-body) {
-    background: white;
-    border: 1px solid #ddd;
+    color: $text-primary;
+    background: $popover-background;
+    border: 1px solid $ui-pane-border;
     padding: 4px;
     border-radius: 4px;
     padding: 4px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)